docs/usage/semantic-commits.md

```diff
@@ -5,40 +5,84 @@ description: Configuring Renovate to use Semantic Commits
 
 # Semantic Commit messages
 
-To detect if your repository uses semantic commits, Renovate looks at the latest 10 commit messages in the base branch.
-It then uses [conventional-commits-detector](https://github.com/conventional-changelog/conventional-commits-detector) to determine what convention the commit messages follow.
+Renovate looks at the last 10 commit messages in the base branch to decide if the repository uses semantic commits.
+If there are semantic commits, Renovate uses the [conventional-commits-detector](https://github.com/conventional-changelog/conventional-commits-detector) to decide what convention the commit messages follow.
 
-Renovate only detects Angular-style conventional commits, it ignores all other commit conventions.
+Renovate only finds Angular-style conventional commits, it ignores other commit conventions.
 
-When Renovate finds Angular-style commits, Renovate will create commit messages and PR titles that look like this:
+When Renovate finds Angular-style commits, Renovate creates commit messages and PR titles like this:
 
 - chore(deps): update eslint to v7.30.0
 - fix(deps): update express to v4.17.1
 
-Renovate uses the `chore` prefix by default.
-When you extend `config:base`, Renovate still defaults to `chore`, but will use the `fix` prefix for npm production dependencies (`devDependencies` still use `chore`).
+By default, Renovate uses the `chore` prefix.
```

Comment:
First we show an example PR title:

> - fix(deps): update express to v4.17.1

And just after that we say:

> By default, Renovate uses the `chore` prefix.

I'm confused about the default behavior now, because the example shows `fix(deps):` while we say Renovate defaults to `chore:`...

Do you have any ideas on how to fix the text or the example, or both?

Reply:
We can simplify to show only the `chore` example